BUILD.gn

```diff
@@ -79,47 +79,165 @@ if (is_fuchsia) {
       "$flutter_root/content_handler:aot",
     ]
     if (flutter_runtime_mode != "release") {
-      deps += [ "//third_party/dart/runtime/observatory:embedded_archive_observatory" ]
+      deps += [
+        "//third_party/dart/runtime/observatory:embedded_archive_observatory",
+      ]
     }
 
     binary = "flutter_aot_runner"
 
     if (flutter_runtime_mode != "release") {
-      resources = [ {
-            path = rebase_path(
-                    "$root_gen_dir/observatory/embedded_archive_observatory.tar")
-            dest = "observatory.tar"
-          } ]
+      resources = [
+        {
+          path = rebase_path(
+                  "$root_gen_dir/observatory/embedded_archive_observatory.tar")
+          dest = "observatory.tar"
+        },
+      ]
     }
 
-    meta = [ {
-          path = rebase_path("content_handler/meta/sandbox")
-          dest = "sandbox"
-        } ]
+    meta = [
+      {
+        path = rebase_path("content_handler/meta/sandbox")
+        dest = "sandbox"
+      },
+    ]
+  }
+
+  package("flutter2_aot_runner") {
+    deps = [
+      "$flutter_root/content_handler:aot",
+    ]
+    if (flutter_runtime_mode != "release") {
```

Comment:
Not really. Currently one can choose between jit ("debug") and aot ("profile"), but that choice is app by app instead of the whole build, and there's no way to get "release". It's future work to add a way for apps to select "release", so for now this is mostly a reminder that a "release" runner shouldn't include the Observatory tar.